src/constants/buyCover.ts

```diff
@@ -0,0 +1,2 @@
+export const COMMISSION_DENOMINATOR = 100_00;
```

Comment:
Is this ten thousand? If so, shouldnt it be `10_000`?

Reply:
good question! tbh, I got it from the cover router previous PR here, to keep consistency. https://github.com/NexusMutual/cover-router/pull/63/files#diff-4948e93b6ff32fd5b5228f82edaceb099d7c5437eae914fe3448126e2ae075d3L22. I think @MilGard91 or @shark0der have more context to answer this question 👀

Reply:
my hunch on this was that commission/slippage is a percent so out of `100%` => `100_00`

Reply:
yes, it was written like this to represent percentages 100.00%